satpy/node.py

```diff
@@ -324,17 +325,22 @@ def _find_compositor(self, dataset_key):
         except KeyError:
             raise KeyError("Can't find anything called {}".format(
                 str(dataset_key)))
-
+        if resolution:
+            compositor.info['resolution'] = resolution
+        if calibration:
+            compositor.info['calibration'] = calibration
+        if polarization:
+            compositor.info['polarization'] = polarization
```

Comment:
@mraspaud The only thing about this PR that feels wrong to me after looking in to this issue more is this set of `.info` assignments. To me this says the compositor must produce something with these properties, but that is not necessarily true. It also seems odd because `calibration` and `resolution` can be lists iirc so this may have unintended results in the compositor. Then again I'm not sure how else someone request the compositor to do a specific resolution so...I guess I'm ok with this. Or maybe the `dataset_key` passed to `get_compositor` should have these things set.